tests/test_obs_img_utils_api.py

```diff
@@ -24,7 +24,7 @@
 from obs_img_utils.exceptions import PackageVersionExceptionOBS
 
 
-class TestAPI(object):
+class TestAPI:
     def setup(self):
```

Comment:
This can be renamed to `setup_method` with no other changes and it will fix the nose issue.

Reply:
I fixed this by adding a setup_method, because if you only rename it, I think it will no longer work for py3.6 (SLE)
but have to double check

Reply:
ok rename works, done. Thanks